src/main/java/com/iexec/worker/tee/post/compute/worker/WorkerApiManager.java

```diff
@@ -36,4 +39,11 @@ public static WorkerApiClient getWorkerApiClient() {
         }
         return workerApiClient;
     }
+
+    public static String getWorkerHost() {
+        final String host = EnvUtils.getEnvVar(WORKER_HOST_ENV_VAR);
+        return StringUtils.isBlank(host)
+                ? "worker:13100"
```

Comment:
Can you please 
1. move that static string on top of class?
2. add a simple case for covering `getWorkerHost`?

Reply:
Please see https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/71/commits/f4705ea540492a0f26c3fd7fed74e5ff6e51e5b0.